.travis.yml

```diff
@@ -8,14 +8,9 @@ bundler_args: --without debug release
 before_install:
   - ruby -v | (grep -v "rubinius" && gem install bundler) || echo 0
```

Comment:
`rbx` is gone from matrix, so this can be too

Reply:
Good  catch